Library/Homebrew/test/keg_test.rb

```diff
@@ -44,7 +44,9 @@ def teardown
     rmtree HOMEBREW_PREFIX/"bin"
     rmtree HOMEBREW_PREFIX/"lib"
   end
+end
 
+class LinkTests < LinkTestCase
```

Comment:
This isn't a cleanup for the sake of it. The tests fail if I don't do this. That's how I discovered #1526!